advisor/2016-07-12-preview/swagger/advisor.json

```diff
@@ -0,0 +1,461 @@
+{
```

Comment:
If this belongs to ARM plane then swagger must be placed in `arm-advisor` folder in the repo otherwise the location is perfect :)